src/server/_printer.py

```diff
@@ -203,6 +210,9 @@ def _format_row(self, first: bool, row: Dict):
         if first:
             self._writer = DictWriter(self._stream, list(row.keys()), lineterminator="\n")
             self._writer.writeheader()
+            if show_hard_api_key_warning() and columns:
```

Comment:
I'm not sure how these printers works, but here we have columns variable that is not defined and it makes this CSVPrinter broken.